api/v1/models/Token.js

```diff
@@ -0,0 +1,38 @@
+var Promise = require('bluebird');
+
+var Model = require('./Model');
+var User = require('./User');
+var Crypto = require('../utils/crypto');
+
+var Token = Model.extend({
+	tableName: 'tokens',
+	idAttribute: 'id',
+	hasTimestamps: ['created'],
+	user: function(){
+		return this.belongsTo(User, 'user_id');
+	},
+	validations: {
+		value: ['required', 'string']
+	}
+});
+
+/**
+ * Finds a token given the Token value
+ * @param {String|Number} value The Token's value
+ * @return {Promise} resolving to the associated Token Model
+ * @throws {NotFoundError} when the requested token cannot be found
```

Comment:
This error is not ever thrown (and should probably be thrown instead in a `TokenService`)